c-pallets/segment-book/src/lib.rs

```diff
@@ -504,8 +513,9 @@ pub mod pallet {
 			Ok(())
 		}
 
+		// Seems ut is unifinished?
 		#[transactional]
-		#[pallet::weight(0)]
+		#[pallet::weight(0)] // fix weight
```

Comment:
This is an unsigned transaction. I'm not sure if I need to calculate weight for it